postgres/postgres.go

```diff
@@ -54,7 +54,7 @@ type Stmt struct {
 }
 
 // Prepare statement
-func (s *Stmt) Prepare(db *sqlx.DB, SQL string) (statement *sql.Stmt, err error) {
+func (s *Stmt) Prepare(db *sqlx.DB, tx *sql.Tx, SQL string) (statement *sql.Stmt, err error) {
```

Comment:
Changed function signature, you need open PR upgrading where you use the adapter

Reply:
This function is only used inside the adapter package.
The other packages in the pRest project do not access it.